src/backend/InvenTree/part/events.py

```diff
@@ -0,0 +1,7 @@
+"""Event definitions and triggers for the part app."""
+
+from generic.events import BaseEventEnum
+
+
+class PartEvents(BaseEventEnum):
+    """Event enumeration for the Part models."""
```

Comment:
Q (blocking): Is this left empty by choice?

Reply:
There are no "part" events yet